cram_pr2/cram_pr2_shopping_demo/launch/world.launch

```diff
@@ -0,0 +1,33 @@
+<launch>
+    <arg name="upload" default="true"/>
+
+    <param name="shelve_description" command="$(find xacro)/xacro.py
```

Comment:
Please use either "shelf" or "shelves", the word "shelve" does not exist :)

Reply:
This applies to all usages of this word: in file names, inside code, in ROS parameters etc.